src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -167,6 +167,30 @@ ClientRequest::interruptible_future<> ClientRequest::with_pg_process_interruptib
       pg.wait_for_active_blocker,
       &decltype(pg.wait_for_active_blocker)::wait));
 
+  if (int res = op_info.set_from_op(&*m, *pg->get_osdmap());
```

Comment:
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/crimson/osd/osd_operations/client_request.cc:173:45: error: member reference type 'crimson::osd::PG' is not a pointer; did you mean to use '.'?
  if (int res = op_info.set_from_op(&*m, *pg->get_osdmap());
```

Reply:
Ah, sorry, I'm on a trip now, and the build runs too slow on my notebook, so I directly pushed the code. Will correct it soon.

Reply:
Done, please take a look, again, thanks:-)